setup.py

```diff
@@ -86,41 +86,52 @@ def get_package_version(package):
             return str(version_info)
 
 
-def check_requirements():
+def check_runtime_requirements():
     if sys.version_info < PYTHON_VERSION:
         raise SystemExit('You need Python version %d.%d or later.' \
                          % PYTHON_VERSION)
     for (package_name, min_version) in DEPENDENCIES.items():
         if package_name == 'cython':
             package_name = 'Cython'
-        dep_error = ''
+
         if package_name.lower() == 'pillow':
             package_name = 'PIL.Image'
             min_version = '1.1.7'
+
+        dep_errors = []
         try:
             package = __import__(package_name,
                 fromlist=[package_name.rpartition('.')[0]])
         except ImportError:
-            dep_error = ('You need `%s` version %s or later.'
-                         % (package_name, min_version))
+            dep_errors.append((package_name, min_version, '--'))
         else:
             if package_name == 'PIL':
                 package_version = package.PILLOW_VERSION
             else:
                 package_version = get_package_version(package)
 
-            if LooseVersion(min_version) > LooseVersion(package_version):
-                dep_error = ('You need `%s` version %s or later,'
-                             'found version %s.'
-                             % (package_name, min_version,
-                                package_version))
-        if dep_error:
-            raise ImportError(dep_error)
+            if not package_version or LooseVersion(min_version) > LooseVersion(package_version):
+                dep_errors.append((package_name, min_version, package_version))
+
+        if dep_errors:
+            print('=========================================================')
+            print('Warning: Unsatisfied dependencies for using scikit-image!')
+            print('=========================================================')
+            for (pkg, version_required, version_installed) in dep_errors:
+                print('You need                ', 'You have'                 )
+                print('---------------------------------------------------------')
+                print(('%s %s' % (pkg, version_required)).ljust(24),
+                      version_installed)
+            print('=========================================================')
+            print('Install runtime dependencies using:')
+            print('')
+            print('  pip install -r requirements.txt')
```

Comment:
I'm not so sure about this.  Wheels are only available for numpy and scipy on OSX.